examples/next/README.md

```diff
@@ -0,0 +1,29 @@
+## Setting up
+In order to try out these examples, you need to set your public API key as the value of the NEXT_PUBLIC_FPJS_PUBLIC_API_KEY variable\
+in the `.local.env`, you can just copy `.env` file, located in this directory.
+
+### FingerprintJS public API key
```

Comment:
I'd remove this section. Whenever it's possible try to link our official docu. I'd use this link here - https://dev.fingerprintjs.com/docs/js-agent#agent-initialization. It'll be easier to maintain.

Reply:
done